src/VisualStudio/Core/Def/Options/VisualStudioOptionStorage.cs

```diff
@@ -364,7 +364,6 @@ public bool TryFetch(LocalUserRegistryOptionPersister persister, OptionKey2 opti
         {"dotnet_format_on_save", new LocalUserProfileStorage(@"Roslyn\Internal\OnOff\Features", "FormatOnSave")},
         {"dotnet_enable_full_solution_analysis_memory_monitor", new LocalUserProfileStorage(@"Roslyn\Internal\OnOff\Features", "Full Solution Analysis Memory Monitor")},
         {"dotnet_code_analysis_in_separate_process", new LocalUserProfileStorage(@"Roslyn\Internal\OnOff\Features", "OOP64Bit")},
-        {"dotnet_enable_core_clr_in_code_analysis_process", new LocalUserProfileStorage(@"Roslyn\Internal\OnOff\Features", "OOPCoreClr")},
```

Comment:
We have integration test depends on this, we should remove those CI legs as part of the change

[https://github.com/dotnet/roslyn/blob/02888a4571bfb8ce33f29df4a0504c59be68ca5e/eng/build.ps1#L629](https://github.com/dotnet/roslyn/blob/02888a4571bfb8ce33f29df4a0504c59be68ca5e/eng/build.ps1#L629)

Reply:
will do.